clustering/web/extension/src/main/resources/subsystem-templates/distributable-web.xml

```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--  See src/resources/configuration/ReadMe.txt for how the configuration assembly works -->
+<config>
+    <extension-module>org.wildfly.extension.clustering.web</extension-module>
+    <subsystem xmlns="urn:jboss:domain:distributable-web:1.0" default-session-management="default" default-single-sign-on-management="default">
+        <infinispan-session-management name="default" cache-container="web" granularity="SESSION" routing="OWNER"/>
```

Comment:
This is pretty minor, the order of the xml attributes generated by galleon is not defined, but it seems it's always the same, would you mind using this order here?
`<infinispan-session-management name="default" granularity="SESSION" routing="OWNER" cache-container="web"/>`
That matches with the one generated by galleon. It will reduce differences based on the position of the attributes when we are comparing legacy-builds and builds done via galleon.

Reply:
:-1:  Given that the attribute ordering in galleon is undefined - any change to the set of attributes will likely completely change the ordering within xml.  I don't think it is realistic to ask each subsystem maintainer to keep these in sync.